R/pkg/R/DataFrame.R

```diff
@@ -1231,6 +1296,24 @@ setMethod("unionAll",
             dataFrame(unioned)
           })
 
+setGeneric("rbind", signature = "...")
+
+rbind.SparkDataFrames <- function(x, ..., deparse.level = 1) {
+  allargs <- list(...)
+  if (nargs() == 3) {
+    unionAll(x, ...)
+  } else {
+    unionAll(x, Recall(..., deparse.level = 1))
+  }
+}
+
+#' @rdname rbind
+#' @aliases unionAll
+setMethod("rbind",
+          signature(... = "DataFrame"),
+          rbind.SparkDataFrames
```

Comment:
Sure.
BTW: what do you think about letting unionAll accept more than two DataFrames?